Core/include/Acts/EventData/VectorMultiTrajectory.hpp

```diff
@@ -0,0 +1,327 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/EventData/MultiTrajectory.hpp"
+
+#include <unordered_map>
+
+namespace Acts {
+
+class VectorMultiTrajectory final
+    : public MultiTrajectory<VectorMultiTrajectory> {
+  friend MultiTrajectory<VectorMultiTrajectory>;
```

Comment:
The `*_impl` functions are `private`, but I have to call them from the base class. I think it would be the same for `protected`. 
Can you think of another way to do this?

Reply:
Ah, missed the "private". All ok.